common/test-setting-immutable-prototype.js

```diff
@@ -9,14 +10,21 @@ self.testSettingImmutablePrototypeToNewValueOnly =
   let dunderProtoError = "SecurityError";
   let dunderProtoErrorName = "\"SecurityError\" DOMException";
   if (isSameOriginDomain) {
-    dunderProtoError = new TypeError();
+    // Is this actually right?  Or should this TypeError be thrown in our
+    // global?
```

Comment:
Both Chrome and Firefox pass the relevant tests it seems, so yes?

Reply:
Ah, sorry, I forgot I left that in.  I just rechecked, and this is in fact right given that `__proto__` is an accessor, so we are entering the Realm of the target here.  I'll remove that comment.